fbw-a32nx/src/systems/instruments/src/EWD/shared/EwdSimvarPublisher.tsx

```diff
@@ -148,17 +148,17 @@ export class EwdSimvarPublisher extends SimVarPublisher<EwdSimvars> {
         ['engine1FF', { name: EwdVars.engine1FF, type: SimVarValueType.Number }],
         ['engine1N1', { name: EwdVars.engine1N1, type: SimVarValueType.Number }],
         ['engine1N2', { name: EwdVars.engine1N2, type: SimVarValueType.Number }],
-        ['engine1Reverse', { name: EwdVars.engine1Reverse, type: SimVarValueType.Bool }],
-        ['engine1ReverseNozzle', { name: EwdVars.engine1ReverseNozzle, type: SimVarValueType.Number }],
+        ['engine1ReverserTransit', { name: EwdVars.engine1ReverserTransit, type: SimVarValueType.Bool }],
+        ['engine1ReverserDeployed', { name: EwdVars.engine1ReverserDeployed, type: SimVarValueType.Bool }],
```

Comment:
is there a reason there needs to be 2 simvars to handle transit and deployed state? Just wondering if its better consolidated into a single simvar with 3 states.